packages/rolldown/tests/fixtures/tree-shake/external-export-star1/main.js

```diff
@@ -0,0 +1 @@
+export * from 'fs'
```

Comment:
Before we would remove this statement since it is considered side effect free if the user specifies `treeshake.moduleSideEffects: false`